docs/_component/editor.client.js

```diff
@@ -234,7 +245,7 @@ export const Editor = ({children}) => {
           <div className="frame-body frame-body-box-fixed-height frame-body-box">
             {state.file && state.file.result ? (
               <ErrorBoundary FallbackComponent={FallbackComponent}>
-                {state.file.result()}
+                <Preview />
               </ErrorBoundary>
```

Comment:
Does this remove the need for `ErrorBoundary`?

Reply:
And isn’t it weird that `ErrorBoundary` doesn‘t catch the error in question? Is it a bug there, or something we should change here to make it work?

Reply:
Seems like ErrorBoundary is unnecessary for Preview here. 

I'm not entirely sure why try...catch can catch the error but ErrorBoundary doesn't either.